nncf/experimental/openvino_native/engine.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
We discussed with @AlexKoff88 that `openvino_native` would be easier to move from the `experimental`. Because we plan to keep `OpenVINO POT` and `OpenVINO Native` backends.

Reply:
The actual sub-backend should be abstracted away from the user, *unless* they want a result as close to POT as possible. This can be handled by a single additional parameter in a function, it seems. I'm not sure that we need to have POT and native approaches as separate backends at the `BackendType` level.